dcs/action.py

```diff
@@ -1787,6 +1787,25 @@ def dict(self):
         return d
 
 
+class PictureToAll(Action):
+    predicate = "a_out_picture"
+
+    def __init__(self, file_res_key: Optional[ResourceKey] = None):
```

Comment:
Is the file key actually optional? That's up to DCS and DCS can sometimes be surprising, but that does surprise me.